spec/models/volunteer_spec.rb

```diff
@@ -68,6 +68,24 @@
       case_contacts.each { |c| c.reload }
       expect(case_contacts).to all(satisfy { |c| !c.active })
     end
+
+    context "when volunteer has previously been assigned a supervisor", runme: true do
+      let!(:supervisor_volunteer) { create(:supervisor_volunteer, volunteer: volunteer) }
+
+      it "deactivates the supervisor-volunteer relationship" do
+        volunteer.deactivate
+
+        expect { volunteer.reload }.to change(volunteer, :supervisor_volunteer)
```

Comment:
I think it's a little weird since what does the change is the `deactivate` method, this way it seems to be the `reload` method instead. WDYT about this:
```suggestion
        expect { volunteer.deactivate }.to change(volunteer, :supervisor_volunteer)
```
?

Reply:
Thank you for the suggestion! I'll look into it!